fractal_server/app/schemas/user.py

```diff
@@ -53,6 +54,19 @@ class UserUpdate(schemas.BaseUserUpdate):
     )
 
 
+class UserUpdateStrict(BaseModel):
+    """
+    Attributes that every user can self-edit
+    """
+
+    cache_dir: Optional[str]
+    password: Optional[str]
+
+    _cache_dir = validator("cache_dir", allow_reuse=True)(
```

Comment:
Question: what happens for `UserUpdateStrict(password="1234")`? is this validator only acting when `cache_dir` is set?